resources/fetchers/monitoring_fetcher.go

```diff
@@ -77,3 +105,26 @@ func (r MonitoringResource) GetMetadata() (fetching.ResourceMetadata, error) {
 	}, nil
 }
 func (r MonitoringResource) GetElasticCommonData() any { return nil }
+
+func (s SecurityHubResource) GetData() any {
+	return s
+}
+
+func (s SecurityHubResource) GetMetadata() (fetching.ResourceMetadata, error) {
+	if s.DescribeHubOutput == nil || s.HubArn == nil {
```

Comment:
Should we send a resource with no output to evaluation?

Reply:
Yes, it means that the security hub was not enabled